lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/traffic/LambdaTrafficUpdateVerificationTask.java

```diff
@@ -63,7 +73,47 @@ public TaskResult execute(@Nonnull StageExecution stage) {
             return formErrorTaskResult(stage,op.getErrors().getMessage());
         }
 
+        if (!DeploymentStrategyEnum.$WEIGHTED.toString().equals(stage.getContext().get("deploymentStrategy"))) {
+            boolean inValid = validateWeights(stage);
+            if (inValid) {
+                formErrorTaskResult(stage, "Could not update weights in time");
+                return TaskResult.builder(ExecutionStatus.TERMINAL).outputs(stage.getOutputs()).build();
+            }
+        }
+
         copyContextToOutput(stage);
         return taskComplete(stage);
     }
+
+    /**
+     * <p>Waits for the weights of your lambda to be moved from 0% to 100% to the new version
+     * so alias - provisioned concurrency can be updated
+     * </p>
+     * @param stage The runtime execution state of a stage.
+     * @return The boolean value if your alias-weights were moved before max time specified in lambdaPluginConfig.cloudDriverRetrieveMaxValidateWeightsTime
```

Comment:
Does this match the code?

Reply:
No, i will change it to `The boolean value if your alias-weights failed to be moved before max time specified in lambdaPluginConfig.cloudDriverRetrieveMaxValidateWeightsTime`